views/guide.lavender

```diff
@@ -76,17 +76,17 @@ block content
                     'lavender/lavender': '*'
                   }
                 }
-            | then run
+            | Then, run
             code.language-markup php composer.phar install 
             | or
-            code.language-markup composer install
-            | depending on how you've installed composer. Once composer has finished 
+            code.language-markup composer install,
+            | depending on how you've installed composer. Once composer has finished, 
```

Comment:
The comma is in a code tag, meaning it will show up in the code being presented to the viewer. It should go in the line below it, like this:

| , depending on how you've installed